### ISSUE TITLE
Diversion from approach flown with VNAV/LNAV

### ISSUE THREAD

**Issue by mikebakke** [Working-Title-MSFS-Mods/fspackages#1004]
The flight correctly tracks SID, en-route, STAR and entry to the approach but flies a circle in the middle of the approach phase. The Image shows the track taken - the one relevant to the video is the one extending furthest to sea. I just let it do it until it was close enough to hit APPR when it recovered.
![crazy-ivan](https://user-images.githubusercontent.com/55141164/110927753-f1442f80-831d-11eb-9e76-75224a7388a5.png)

Video of the whole approach is here: https://streamable.com/tbxh2e

**Comment by mikebakke** [comment]
Watching the video again I see it suddenly discard 84NWC and very shortly after that it drops CI25 and makes the circle in order to get to FI25 which is obviously way late...

**Comment by ourduck** [comment]
EGLC BPK N601 POL.POL1N RNAV25  What's interesting in putting this in to the FMC that I selected RNAV25 UPMOP IAF and I got a discontinuity inside the approach procedure.  There's no vector there is there, its part of the procedure?

![image](https://user-images.githubusercontent.com/80160370/110964453-34ec6880-8321-11eb-893b-a4e79235e275.png)


Here's the plate...

![image](https://user-images.githubusercontent.com/80160370/110964551-4a619280-8321-11eb-8525-cae1ce92a433.png)


Here its a bit clearer...is this a Navigraph issue or a LNAV issue?

![image](https://user-images.githubusercontent.com/80160370/110964703-767d1380-8321-11eb-882f-7059aed8d129.png)

**Comment by mikebakke** [comment]
Hi
Yes it looks strange but I've never used that arrival/approach - always
RIMT1N / ILS25. I normally get a disco after ETSES but just clear it and
the other 99 times I've flown this, it tracked down to the ground hands
off... Something is squirrely somewhere at the moment I think..

**Comment by ourduck** [comment]
I'm on the approach now...how did you program in your offset to cause the turn?  NT085/9 doesn't seem to be taking for me.

**Comment by mikebakke** [comment]
All I did was load in the FP from SB and add ILS25 from RIMT1N with NT
transition - it all goes in just fine. I do have Navigraph data configured

**Comment by mattnischan** [comment]
Currently the CJ4 does not always properly calculate teardrop course reversals, so that's why you're seeing that issue on this approach.

**Comment by mikebakke** [comment]
Hi Matt

ok - I never had this issue previously. Let me roll back a few versions and
retry to see if behaviour changes.

Cheers

**Comment by ourduck** [comment]
![image](https://user-images.githubusercontent.com/80160370/110969238-703d6600-8326-11eb-8107-2be250edd418.png)
![image](https://user-images.githubusercontent.com/80160370/110969274-7b909180-8326-11eb-9665-ba0ead14da01.png)

**Comment by mikebakke** [comment]
Also meant to say the course drawn and normally followed is as you suggest
a triangle - not the smooth curve.

**Comment by mikebakke** [comment]
I've re-run today 4 times on 10.04, 11.0 and 11.1 twice with no repeat of the event. The only thing I noticed was that 
the 2nd run with 11.1 I ran with the approach chart displayed and the final turn was earlier and harder. I did that run because both events yesterday occurred when I had charts displayed. I did wonder if there was any additional processing load for the moving map display which could could impact the calculations for the course but.... Anyway - can't reproduce so I am closing.. 

Cheers

Mike
![4 - test runs - Little Navmap Map 20210313-153155](https://user-images.githubusercontent.com/55141164/111035248-6ba1ac00-8411-11eb-8363-5fc3879f24ae.jpg)
